images/scripts/debian.setup

```diff
@@ -45,6 +44,13 @@ udisks2 \
 udisks2-lvm2 \
 "
 
+if [ "$1" = "debian-testing" ]; then
+    COCKPIT_DEPS="$COCKPIT_DEPS libblockdev-mdraid3"
+else
+    COCKPIT_DEPS="$COCKPIT_DEPS libblockdev-mdraid2"
+fi
```

Comment:
Please flip this around and only include libblockdev-mdraid2 on the "old" OSes (everything but debian-testing), so that this list gradually shrinks to zero. Please don't include libblockdev-mdraid3 at all, udisks >= 2.10 depends on it directly. See https://github.com/cockpit-project/cockpit/pull/19092